client/views/windows/connect.tpl

```diff
@@ -43,6 +43,12 @@
 					Enable TLS/SSL
 				</label>
 			</div>
+			<div class="col-sm-9 col-sm-offset-3">
+				<label class="tls">
+					<input type="checkbox" name="rejectUnauthorized" {{#if defaults.rejectUnauthorized}}checked{{/if}} {{#if lockNetwork}}disabled{{/if}}>
+					Reject unauthorized TLS certificates
```

Comment:
I feel like there is a more idiomatic way to say this but I can't think of it. "Allow untrusted certificates"?
By the way I would go for "Allow" rather than "Reject", it seems to me that it's more common to see this wording, but also because an unchecked checkbox means "Do not reject" double-negation type 😅
No big deal if you feel strongly against though.

(I realize that `irc-framework` calls it `rejectUnauthorized`, but I don't think that's the best wording from an end-user standpoint)

Reply:
`rejectUnauthorized` comes from Node, which is why I kept it. https://nodejs.org/api/tls.html#tls_tls_createserver_options_secureconnectionlistener

Reply:
Yeah I get that from the API/internals point of view, but in the UI/UX, it makes more sense to not use a negated version. It's just confusing right now IMO.

Reply:
I'm fine with changing the label. `Verify TLS certificate trust chain`?